src/sysinfo.cc

```diff
@@ -13,13 +13,19 @@
 // limitations under the License.
 
 #include "sysinfo.h"
+#include "internal_macros.h"
 
+#ifndef OS_WINDOWS
```

Comment:
consider flipping this so the positive is first:

#ifdef OS_WINDOWS
# include <Shlwapi.h>
# include <Windows.h>
#else
....
#endif  // OS_WINDOWS